portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,60 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <meta name="description" content="Kenneth's Personal Website">
+    <meta name="keywords" content="Kenneth, Kiprotich">
+    <meta name="author" content="Kenneth Kiprotich">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio: Kenneth Kiprotich</title>
     <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:300,400">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
+      <h1>My Portfolio </h1>
+      <h2>About Me</h2>
+      <p> I am Kenneth Kiprotich, a student Columbia Univerity, studying Computer Science.
+          I am interested in drawing and I enjoy early morning runs. I love eating pizza. </p>
+      <div class="image-thumbs">
+          <a href="images/KennethKiprotich.jpg"><img src="images/KennethKiprotich.jpg"/></a>
+          <!-- <a href="images/stanley-1.jpg"><img src="images/stanley-1.jpg"/></a> -->
```

Comment:
do you still need this?

Reply:
Still working on the particular details of the site.Thanks for catching that. I deleted the unnecessary tag.